ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/ImportOrderingRule.kt

```diff
@@ -23,10 +25,17 @@ class ImportOrderingRule : Rule("import-ordering") {
             val children = node.getChildren(null)
             if (children.isNotEmpty()) {
                 val imports = children.filter { it.elementType == IMPORT_DIRECTIVE }
+                val hasComments = children.find { it.elementType == BLOCK_COMMENT || it.elementType == EOL_COMMENT } != null
```

Comment:
a broad brush. should we check for the suppression message? though perhaps unlikely to be a problem in practice.

Reply:
The suppression message causes a real problem, but it would also seem annoying if you had comments (for some reason?) in an import list that they'd get blown away by the autoformatter.

Reply:
true. sounds good.